docs/en/estimation/mview_mperson_end2end_estimator.md

```diff
@@ -0,0 +1,77 @@
+# Multi-view Multi-person End-to-end Estimator
+
+- [Overview](#overview)
+- [Arguments](#arguments)
+- [Run](#run)
+  - [Step0: estimate keypoints3d](#step0-estimate-keypoints3d)
+  - [Step1: optimize keypoints3d](#step1-optimize-keypoints3d)
+  - [Step2: estimate smpl](#step2-estimate-smpl)
+- [Example](#example)
+
+## Overview
+
+This end-to-end estimator tool takes multi-view RGB sequences and multi-view calibrated camera parameters as input. By a simple call of `run()`, it outputs keypoints3d predicted by the model trained with learning based method in an end-to-end manner, as well as SMPL parameters for the multi-person in the multi-view scene.
+
+## Arguments
+
+- **output_dir**:
+`output_dir` is the path to the directory saving all possible output files, including keypoints3d, SMPLData and visualization videos.
+- **model_dir**:
+`model_dir` is the path the the pretrained model for keypoints3d inference.
```

Comment:
two 'the' between 'path' and 'pretrained model'

Reply:
Fixed in [c2ba151](https://github.com/openxrlab/xrmocap/pull/46/commits/c2ba1517e1c5c68dd276bf21e002f6e4ef4df715).